scrapers/covid-19/historical.js

```diff
@@ -66,6 +74,20 @@ const historicalV2 = async (keys, redis) => {
 			newElement.timeline.cases[timelineKey[i]] = parseInt(cases[i]);
 			newElement.timeline.deaths[timelineKey[i]] = parseInt(deaths[i]);
 			newElement.timeline.recovered[timelineKey[i]] = parseInt(recovered[i] || 0);
+
+			/*
+			This sets cases and deaths of the timeline field to 0 and populates
+			recovered field with aggregate recovered data for Canada. This ensures
+			conformity with other real provinces of Canada. This is a fake province
+			which is holding aggregate recovered data for Canada. This has to be done
+			here because the original data values are strings which need to be parsed
+			to integers.
+			*/
+			if (index === 0) {
```

Comment:
Why if `index === 0`? This seems out of place here, if you're just trying to do this once, do it where you initialize `timeline: { cases: {}, deaths: {}, recovered: {}` ie
```js
for (let i = 0; i < cases[0].length; i++) { ...
```

Reply:
> Why if index === 0? This seems out of place here, if you're just trying to do this once, do it where you initialize timeline: { cases: {}, deaths: {}, recovered: {} ie

You are right. It is `index === 0` because  I want to do it once. The aggregate recovered data  have string values. Essentially I want to take advantage of the loop to parse the string values into integers as I explained in the preceding comment.  If you want it removed, then I can take outside the callback.

Reply:
Ok cool, lets move it out